src/package.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Why this?

Reply:
https://github.com/facebook/react-native/issues/3099#issuecomment-221815006

Any other way to do absolute path in react native?

Reply:
understood. Not sure about the other way.